autoPyTorch/pipeline/tabular_classification.py

```diff
@@ -216,6 +246,7 @@ def _get_pipeline_steps(self, dataset_properties: Optional[Dict[str, Any]],
             ("feature_preprocessor", FeatureProprocessorChoice(default_dataset_properties)),
             ("tabular_transformer", TabularColumnTransformer()),
             ("preprocessing", EarlyPreprocessing()),
+            ("network_embedding", NetworkEmbeddingChoice(default_dataset_properties)),
```

Comment:
I guess this is also needed in regression?

Reply:
yeah thanks for pointing out, I think I forgot